demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -999,6 +1027,184 @@ def _collect_widget_dependencies(pack_widgets: list, id_set: dict, verbose: bool
 
         return dependencies_packs
 
+    @staticmethod
+    def _collect_generic_types_dependencies(pack_generic_types: list, id_set: dict, verbose: bool,
+                                            exclude_ignored_dependencies: bool = True) -> set:
+        """
+        Collects generic types dependencies.
+        Args:
+            pack_generic_types (list): collection of pack generics types data.
+            id_set (dict): id set json.
+            verbose (bool): Whether to log the dependencies to the console.
+            exclude_ignored_dependencies (bool): Determines whether to include unsupported dependencies or not.
+        Returns:
+            set: dependencies data that includes pack id and whether is mandatory or not.
+        """
+        dependencies_packs = set()
+        if verbose:
+            click.secho('### Generic Types', fg='white')
+
+        for generic_type in pack_generic_types:
+            generic_type_data = next(iter(generic_type.values()))
+            generic_type_dependencies = set()
+
+            related_playbooks = generic_type_data.get('playbooks', [])
+            packs_found_from_playbooks = PackDependencies._search_packs_by_items_names(
+                related_playbooks, id_set['playbooks'], exclude_ignored_dependencies)
```

Comment:
we need to make sure but I think generics will not be running playbooks - @asafshen am I right here?

Reply:
yes, I think so
generic objects are more similar to indicators (i.e. platform will run a job that will query objects and run it)

but we should validate that with the rest of team working on it

for now, lets consider it as not (at least platform does not consider it as such)

Reply:
Removed